### PR TITLE
feature(unocss): add unocss support

### PR DESCRIPTION
**Pull request by fernandolguevara**
add support for unocss 

[svelte-add-unocss](https://github.com/fernandolguevara/svelte-add-unocss)

### CODE REVIEW

**Comment by fernandolguevara** [comment]
@babichjacob can u take a look to this PR?

**Comment by babichjacob** [comment]
It seems like it was missing the `virtual:uno.css` import in the main file (`src/routes/__layout.svelte` for SvelteKit and `src/main.js` or `src/main.ts` for Vite), and when the UnoCSS config was imported in the Vite config that it was missing the `.js` extension it needed to work. 

Are you ready to transfer the repository to me (so I transfer it to the `svelte-add` organization) and I invite you as a collaborator so we (/ I --- when I have time) can fix it?

About presets --- is it common to use more than one? Should we compact all the booleans into a select / multiselect instead of the extensive questionnaire?

File: projects/svelte-add/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const exit = (text) => {
  * @typedef {"coffeescript" | "javascript" | "typescript"} Script
  * @typedef {"css" | "postcss" | "scss"} StyleLanguage
  * @typedef {"bootstrap" | "bulma" | "tailwindcss"} StyleFramework
- * @typedef {"3d" | "graphql-server" | "mdsvex" | "routify"} Other
+ * @typedef {"3d" | "graphql-server" | "mdsvex" | "routify" | "unocss"} Other
```
Comment (babichjacob) [inline projects/svelte-add/index.js:20]: We'll move this to style framework after merging

**Review by babichjacob — APPROVED** [review]

**Comment by babichjacob** [comment]
@fernandolguevara do you still want this in? The [transfer process is described here](https://github.com/svelte-add/svelte-add/issues/23#issuecomment-867833017).

**Comment by aewing** [comment]
@fernandolguevara why the downvote and close? I was hoping to see this added.

**Pull request closed without merging**